### PR TITLE
Localization instructions

### PR DESCRIPTION
**Pull request by lucperkins**
In response to issue #8704

### CODE REVIEW

File: content/en/docs/home/contribute/localization.md [hunk 1]
```diff
@@ -8,47 +8,87 @@ approvers:
...
+
+The Kubernetes documentation is currently available in the following languages:
+
+{{< languages-list >}}
 
-All l10n work must be stored and tracked within the [Kubernetes organization](https://github.com/kubernetes).
+## Workflow  
 
-### Basis for localizations
+All Kubernetes documentation for all languages is built from the [kubernetes/website](https://github.com/kubernetes/website) repository on GitHub. You don't need to create or maintain a separate repository to add a localization.
```
Comment (tengqm) [inline content/en/docs/home/contribute/localization.md]: AFAICT, this is not the current agreement. We have some translations forked from the English site and volunteers to focus on creating and reviewing translations. The outputs, however, are supposed to be synced back the the kubernetes/website repo. This is to reduce noises on the "main" site and to have l10n contributors more focused on their jobs.

File: content/en/docs/home/contribute/localization.md [hunk 2]
```diff
@@ -8,47 +8,87 @@ approvers:
...
-A l10n team will have a repository specifically dedicated to its work, for example: [kubernetes/kubernetes-docs-cn](https://github.com/kubernetes/kubernetes-docs-cn).
+```bash
+$ git clone https://github.com/kubernetes/website
+$ cd website
+```
 
-{{< note >}}
-**Note:** To open a l10n repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
-{{< /note >}}
+### Releases
```
Comment (tengqm) [inline content/en/docs/home/contribute/localization.md]: This is ambiguous. It reads like the section is about how to release the translated artifacts.  Maybe change it to something like "Selecting Kubernetes Releases".

File: content/en/docs/home/contribute/localization.md [hunk 3]
```diff
@@ -8,47 +8,87 @@ approvers:
...
+contentDir = "content/de"
+weight = 3
+```
 
-### Project
+When assigning a `weight` parameter, make sure to see which of the current languages has the highest weight and simply add 1.
+
+### New directory
+
+In order to begin adding documentation for a new localization, you'll need to add a subdirectory to the
```
Comment (tengqm) [inline content/en/docs/home/contribute/localization.md]: Incomplete sentence.

File: content/en/docs/home/contribute/localization.md [hunk 4]
```diff
@@ -8,47 +8,87 @@ approvers:
...
+
+In the configuration example [above](#configuration), for example, the two-letter code for German is `de`. That means that a `de` directory needs to be added to the `content` directory:
```
Comment (tengqm) [inline content/en/docs/home/contribute/localization.md]: remove "for example, "

Comment (lucperkins) [reply content/en/docs/home/contribute/localization.md]: Please note that this is a WIP and not yet ready for review

File: content/en/docs/home/contribute/localization.md [hunk 5]
```diff
@@ -8,77 +8,85 @@ approvers:
...
-Upstream contributions are welcome and encouraged!
+```bash
+mkdir content/de
+```
 
-For the sake of efficiency, limit upstream contributions to a single pull request per week, containing a single squashed commit.
+## Translating documents
 
-## Output
+We understand that localizing *all* of the Kubernetes documentation would be an enormous task. We're okay with localizations smarting small and expanding later.
```
Comment (tengqm) [inline content/en/docs/home/contribute/localization.md]: smarting -> starting?

File: content/en/docs/home/contribute/localization.md [hunk 6]
```diff
@@ -8,83 +8,136 @@ approvers:
...
+
+```shell
+git clone https://github.com/kubernetes/website
+cd website
+git checkout {{< release-branch >}}
+```
 
 {{< note >}}
-**Note:** To open a l10n repository, [contact the SIG docs lead](https://kubernetes.slack.com/messages/C1J0BPD2M) on Slack for assistance.
+**Note**: There is currently no German translation of the Kubernetes documentation, but you're welcome to create one using the instructions outlined here!
```
Comment (zacharysarah) [inline content/en/docs/home/contribute/localization.md]: I'd remove this note and replace the related content on line 60 with a newline/this comment.

**Comment by zacharysarah** [comment]
Thanks for tackling this, @lucperkins. 

I have some thoughts about further progress about what we should ask for/provide in terms of localization. They're best for a follow-up PR and shouldn't hold up this PR, which is almost ready to go. 

The more I think about branching strategy, the more I think we need to provide some guidance to localization teams. I'm not sure what exact advice to provide, but I do think we need to talk about release branches, choosing a working branch, and choosing a branch (maybe standard across all localization repos?) to identify for periodic merging into k/website#master.

File: content/en/docs/home/contribute/localization.md (hunk 6 above)
Comment (lucperkins) [reply content/en/docs/home/contribute/localization.md]: Good catch! Done

**Comment by chenopis** [comment]
Is this good to go then?

**Pull request merged**